### ISSUE TITLE
querying instant exchange orders triggers the rate limit

### ISSUE THREAD

**Issue by dreacot** [crypto-power/cryptopower#193]
we seem to be query the cex api's too frequently

```
2023-10-17 14:06:53.122 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:06:53.122 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:07.997 [INF] BTC: Current sync progress update is on block 812661, target sync block is 812661
2023-10-17 14:07:13.793 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:13.793 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:34.026 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:34.026 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-10-17 14:07:49.027 [ERR] SHWL: Error syncing exchange server: failed to sync exchange server [{flypme {false    }}] after 3 attempts
2023-10-17 14:07:49.027 [ERR] UI: Error syncing instant swap: failed to sync exchange server [{flypme {false    }}] after 3 attempts
```

**Comment by dreacot** [comment]
i noticed this while reviewing https://github.com/crypto-power/cryptopower/pull/177

**Comment by ukane-philemon** [comment]
We can utilize an exchange rate cache and WebSocket connection(if available) like in #177. But to keep the scope small, this can be implemented in another PR.

**Comment by dreacot** [comment]
Another PR, is fine, but it should follow immediately after #177

It should be better for 177, to leave out exchange rate query for the cex pages, i think that would be better, rather that introducing a regression into the codebase.

**Comment by ukane-philemon** [comment]
> It should be better for 177, to leave out exchange rate query for the cex pages

Uhmm, can you shade more light on this?

**Comment by dreacot** [comment]
> > It should be better for 177, to leave out exchange rate query for the cex pages
> 
> Uhmm, can you shed more light on this?

It seems the centralized exchange pages were modified to work with the websockets? And that's what causing the cex apis to reach reate limit

**Comment by ukane-philemon** [comment]
> It seems the centralized exchange pages were modified to work with the websockets? And that's what causing the cex apis to reach reate limit

Oh, now I understand. Yes, the page is refreshed and since caches are not yet implemented for CEX we end up with multiple requests.

I'll pull the plug for the CEX create order page.

**Comment by dreacot** [comment]
still getting this as of today

```
2023-11-01 11:34:59.220 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-11-01 11:34:59.220 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-11-01 11:35:19.381 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-11-01 11:35:19.381 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-11-01 11:35:39.550 [ERR] SHWL: Error getting order info: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-11-01 11:35:39.550 [ERR] SHWL: Error checking for exchange updates: instantSwap.GetOrderInfo: exchangeclient:error:429 Too Many Requests
2023-11-01 11:35:54.550 [ERR] SHWL: Error syncing exchange server: failed to sync exchange server [{flypme {false    }}] after 3 attempts
2023-11-01 11:35:54.550 [ERR] UI: Error syncing instant swap: failed to sync exchange server [{flypme {false    }}] after 3 attempts
```
